maquetado/index.html

```diff
@@ -9,6 +9,21 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1>
+    <div class="row">
+      <h1 class="book-title bold height"> Título del libro </h1>
```

Comment:
no empty spaces between text and <>, change everywhere this happens:
```suggestion
      <h1 class="book-title bold height">Título del libro</h1>
```